godot-macros/src/class/data_models/field_var.rs

```diff
@@ -13,19 +13,68 @@ use crate::class::{
     FuncDefinition,
 };
 use crate::util::make_funcs_collection_constant;
-use crate::util::KvParser;
+use crate::util::{bail, KvParser};
 use crate::{util, ParseResult};
 
 /// Store info from `#[var]` attribute.
 #[derive(Clone, Debug)]
 pub struct FieldVar {
     pub getter: GetterSetter,
     pub setter: GetterSetter,
+    pub notify: Option<Ident>,
```

Comment:
This still uses "notify" terminology, although the semantics have changed to "general_set".

To be clear, I don't think we should provide both. Notify should be implementable via general-set, otherwise we're not designing the latter powerful enough.